.travis.yml

```diff
@@ -237,7 +237,9 @@ matrix:
       language: java
       jdk: openjdk8
       install:
+        - docker version
         - . ./ci/travis/ci.sh init RAY_CI_LINUX_WHEELS_AFFECTED,RAY_CI_JAVA_AFFECTED,RAY_CI_STREAMING_JAVA_AFFECTED
+        - docker version
       before_script:
```

Comment:
Is this just debug?

Reply:
Depends what you mean. Obviously the purpose of this is to know what version of docker was present before we installed anything, and to know what version of docker we got when we upgraded. We don't *have* to keep this forever, but if the build later breaks due to the underlying environment changing, I suspect this might help to more quickly figure out what happened.

Reply:
Fair enough--The output is already relatively verbose and these really aren't that big :)